pyEpiabm/pyEpiabm/routine/file_population_config.py

```diff
@@ -83,9 +84,14 @@ def make_pop(input_file: str, random_seed: int = None, time: float = 0):
         # Store current cell
         current_cell = None
         # Iterate through lines (one per microcell)
-        for _, line in input.iterrows():
+        for line in input.itertuples():
+            line = line._asdict()
+            # Converting from float to string
+            cell_id_csv = str(line["cell"])
```

Comment:
This is a method on named tuple, and though I agree this is normally bad practice, it seems that it is only underscored so that it does not clash with a potential field name. But if you'd prefer I can change it

Reply:
I think the suggestions in the first comment would be better practice if they work, or do they throw errors? I haven't come across this issue before, but can investigate this locally. I can't find these references to `_fields` you mention - it doesn't seem to be on that line? https://github.com/SABS-R3-Epidemiology/epiabm/blob/7d8b8b08439fa2149e549eb5a46fb348324ce5c0/pyEpiabm/pyEpiabm/routine/file_population_config.py#L132

Reply:
I just mean that if I keep it as a named tuple (and do not use _asdict()) then I would have to change line 132 to read
`if (('household_number' in line._fields) and`

Reply:
So if I do or do not use _asdict(), I will still have at least one line in the code with a function/attribute being accessed with a leading underscore

Reply:
Hmm I know this is annoying but I do want to avoid using these private methods if we can. The pythonic alternative for L132 would be `if hasattr(line, 'household_number') and` - would you mind checking that

Reply:
Done, I have made the changes